recipes/IWSLT22_lowresource/data_proc/to_json.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+"""
+Tamasheq-French data processing.
```

Comment:
we will have to use the same naming convention that usual for this script i.e iwslt22_prepare.py